src/views/docs-view/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react'
```

Comment:
@ashleemboyer This was based on my rough interpretation of [the component organization section in the README](https://github.com/hashicorp/dev-portal#component-organization) - felt like this fit better as `Componentry which represents a full site "view."` rather than `Shareable, smaller components`, and felt like the term `page` is usually applied to a specific route while `view` is applied to components that get rendered on multiple pages.

Wasn't aware of the plan to move server code into the `components` folder - happy to move this back into `components/docs-page` if my interpretation is off 👍

Reply:
Thank you for elaborating! 💖

It should be possible to move that server code to the new home in `views`, right? If so I think that sounds okay! As long as we're able to continue moving code over from the react-components `DocsPage`, then I'm good with these changes. 😎

Reply:
Great Q, I think there should be no problem moving server code into `views` 👍 I don't think NextJS has any special treatment for `src/components` or `src/views` (Or really anything much other than `src/pages`?) so I believe we have a great deal of flexibility here 💪 🎉

Reply:
Sweet! Let's do it ⚡